README.md

```diff
@@ -52,6 +52,51 @@ steps:
       version: 11.0.3
 ```
 
+### `@gravitational/teleport-actions/auth`
+
+`auth` uses Teleport Machine ID to generate a set of credentials which can be
+used with other Teleport client tools such as `tsh` and `tctl`.
+
+The action has the following outputs:
+
+- `identity-file`: the path to the identity file which can be used with `tctl` and `tsh`.
+
+Pre-requisites:
+
+- Teleport 11 or above must be used.
+- Teleport binaries must already be installed in the job environment.
+- You must have created a bot and created a GitHub join token that allows that
+  bot to join.
+- A Linux based runner.
+
+Example usage:
+
+```yaml
+steps:
+  - name: Install Teleport
+    uses: gravitational/teleport-actions/setup@v1
+    with:
+      version: 11.0.3
+  - name: Authorize against Teleport
+    id: auth
+    uses: gravitational/teleport-actions/auth@v1
+    with:
+      # Specify the publically accessible address of your Teleport proxy.
+      proxy: tele.example.com:443
+      # Specify the name of the join token for your bot.
+      token: my-github-join-token
```

Comment:
Just to be clear to any folk that skips the comment above, please append the `-name` otherwise, someone would just place the token here 

Or rename it to something that cannot be interpreted as the tbot token itself.

Reply:
I think this is a bit of a tough situation - we already use `token` as the input in `gravitational/teleport-actions/auth-k8s` and I'm a little worried about inconsistency. I could correct it in both, but that'd be a breaking change...  I'll sleep on this and work out what I want to do.

Reply:
An intermediate solution might be to change the `token: my-github-join-token` to `token: my-github-join-token-name`

Reply:
👍  That sounds less breaking and a good idea.

Reply:
4619fc5